RecoParticleFlow/PFProducer/plugins/importers/GeneralTracksImporterWithVeto.cc

```diff
@@ -134,7 +134,7 @@ importToBlock( const edm::Event& e,
   }
   // now we actually insert tracks, again tagging muons along the way
   reco::PFRecTrackRef pftrackref;  
-  reco::PFBlockElementTrack* trkElem = NULL;
+  reco::PFBlockElementTrack* trkElem = nullptr;
```

Comment:
can this be changed to a smart pointer instead?

Reply:
@slava77 the value here is emplaced into a vector of smart pointers, so it is managed properly when it's used.

I'm trying to make minimal changes given the press of time.